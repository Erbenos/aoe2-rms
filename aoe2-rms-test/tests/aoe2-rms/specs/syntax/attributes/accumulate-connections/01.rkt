#lang aoe2-rms
; Create a wide gap through a forest and then run a road through the created gap.

<LAND-GENERATION>
(base-terrain 'FOREST)
(create-player-lands
 (terrain-type 'FOREST)
 (other-zone-avoidance-distance 10))

<CONNECTION-GENERATION>
(accumulate-connections)
(create-connect-all-lands
 (replace-terrain 'FOREST 'LEAVES)
 (terrain-size 'FOREST 10 0))
(create-connect-all-lands
 (replace-terrain 'LEAVES 'ROAD)
 (terrain-size 'LEAVES 1 0))
