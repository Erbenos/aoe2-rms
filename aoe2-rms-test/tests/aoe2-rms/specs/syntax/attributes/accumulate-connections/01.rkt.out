<LAND_GENERATION>
base_terrain FOREST
create_player_lands {
  terrain_type FOREST
  other_zone_avoidance_distance 10
}
<CONNECTION_GENERATION>
accumulate_connections
create_connect_all_lands {
  replace_terrain FOREST LEAVES
  terrain_size FOREST 10 0
}
create_connect_all_lands {
  replace_terrain LEAVES ROAD
  terrain_size LEAVES 1 0
}
